.github/workflows/build_and_push_image_on_tag.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: "Build and push on tag event"
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on tag events events but only for the master branch
+  push:
+    tags:
+      - '*'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Doesn't this field need subfields?
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
The field does not need any subfields. not neccessary. 
it is useful when we dispatch job with parameters